### PR TITLE
[PFT-1885] export pimlico and remove parifi relayers code

### PR DESCRIPTION
**Pull request by Akhilleshgoswami**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -21,7 +20,6 @@ export class ParifiSdk {
   core: Core;
   relayer: {
     gelato: Gelato;
```
Comment (sudeepb02) [inline src/index.ts:22]: Need to add Pimlico relayer here

**Review by sudeepb02 — CHANGES_REQUESTED** [review]

**Review by sudeepb02 — CHANGES_REQUESTED** [review]
@Akhilleshgoswami Test cases are still failing for me.

File: src/common/subgraphMapper.ts [hunk 2]
```diff
@@ -151,7 +151,19 @@ export const mapOrdersArrayToInterface = (
     throw error;
   }
 };
-
+export const mapOrderArrayToPriceid = (response:any) =>{
+  if (response === null) return undefined;
```
Comment (racso2609) [inline src/common/subgraphMapper.ts]: is better evaluate ```if(!response)``` because we remove more each cases like possible undefined and things like that

File: src/common/subgraphMapper.ts [hunk 3]
```diff
@@ -151,7 +151,19 @@ export const mapOrdersArrayToInterface = (
...
+  try {
+    return response.orders.map((order: Order) => {
+      return  {
+        priceIds :order.market?.feedId
+      }
+    });
+  } catch (error) {
+    console.log('Error while mapping data', error);
```
Comment (racso2609) [inline src/common/subgraphMapper.ts:163]: we dont need this try/catch because we are only printing the error but anyway we are throwing and the error is visible on the implementor size.

File: test/common/constants.ts [hunk 4]
```diff
@@ -1,27 +1,27 @@
 // Test values for ARBITRUM MAINNET
-export const TEST_ORDER_ID1 = 'CLOSEDORD170141183460469231731687303715884105727-100-0';
-export const TEST_ORDER_ID2 = 'CLOSEDORD170141183460469231731687303715884105727-7000--1';
-export const TEST_ORDER_ID3 = 'CLOSEDORD170141183460469231731687303715884105728-100--1';
-export const TEST_ORDER_ID4 = 'CLOSEDORD170141183460469231731687303715884105728-200--1';
-
-export const TEST_POSITION_ID1 = 'CLOSED-POS170141183460469231731687303715884105727-100-1';
+export const TEST_ORDER_ID1 = 'CLOSEDORD13888427117130454197-100-1';
+export const TEST_ORDER_ID2 = '0x7dbfede3fb67992ceefab54d8d485278d7cd205a5d3c3de99c628651b5f88b32';
```
Comment (racso2609) [inline test/common/constants.ts]: why we change this constants? the 0x are not valid snx ordersIds

File: test/pyth-tests/pyth.test.ts [hunk 5]
```diff
@@ -47,11 +47,11 @@ describe('Pyth tests', () => {
   it('should return price ids from subgraph', async () => {
     const parifiSdk = await getParifiSdkInstanceForTesting();
 
-    const orderIds = [TEST_ORDER_ID1, TEST_ORDER_ID2, TEST_ORDER_ID3];
+    const orderIds = [TEST_ORDER_ID1];
```
Comment (racso2609) [inline test/pyth-tests/pyth.test.ts]: revert this change modifying the orderIds shoukld be enought to fix the test

File: test/pyth-tests/pyth.test.ts [hunk 6]
```diff
@@ -47,11 +47,11 @@ describe('Pyth tests', () => {
...
 
     const priceIds: string[] = await parifiSdk.subgraph.getPythPriceIdsForOrderIds(orderIds);
     console.log('priceIds from fn: ', priceIds);
-
+    console.log("priceIds",priceIds)
```
Comment (racso2609) [inline test/pyth-tests/pyth.test.ts:54]: no needed console

**Review by racso2609 — CHANGES_REQUESTED** [review]

File: test/common/constants.ts (hunk 4 above)
Comment (Akhilleshgoswami) [reply test/common/constants.ts]: with the latest  subgraph v1.013 it's the order is we are using

**Review by sudeepb02 — APPROVED** [review]
